### PR TITLE
Eigenvalue Analysis

### PR DESCRIPTION
**Pull request by rickyaristio**
Reintroduce sub-panel for eigenvalue analysis. Also, there are some updates related to the Eigenvalue analysis.

### CODE REVIEW

File: Cocodrilo/Cocodrilo_GH/PreProcessing/Analysis/EigenvalueAnalysis_GH.cs [hunk 1]
```diff
@@ -57,7 +59,40 @@ protected override void SolveInstance(IGH_DataAccess DA)
...
+        {
+            writer.SetString("EigenSolverType", mEigenSolverType);
+            return base.Write(writer);
+        }
+
+        public override bool Read(GH_IO.Serialization.GH_IReader reader)
+        {
+            // int eigen_solver_type_index = -1;
+            // if (reader.TryGetInt32("EigenSolverType", ref eigen_solver_type_index))
+            //     mEigenSolverType = (string)eigen_solver_type_index;
```
Comment (tteschemacher) [inline Cocodrilo/Cocodrilo_GH/PreProcessing/Analysis/EigenvalueAnalysis_GH.cs:93]:
```suggestion
```

Do we need this commented code still?

**Comment by tteschemacher** [comment]
Hey @rickyaristio 

Thanks for the change and thanks a lot for keeping things up to date!! I recall that last time the issue was the designer. So it should be good this time.

File: Cocodrilo/Cocodrilo_GH/PreProcessing/Analysis/EigenvalueAnalysis_GH.cs (hunk 1 above)
Comment (rickyaristio) [reply Cocodrilo/Cocodrilo_GH/PreProcessing/Analysis/EigenvalueAnalysis_GH.cs:93]: > Hey @rickyaristio
> 
> Thanks for the change and thanks a lot for keeping things up to date!! I recall that last time the issue was the designer. So it should be good this time.

Yeah, in this PR I recreate the eigenanalysis panel, so there is no issue with the designer this time.

Comment (rickyaristio) [reply Cocodrilo/Cocodrilo_GH/PreProcessing/Analysis/EigenvalueAnalysis_GH.cs:93]: > Do we need this commented code still?

Ups, I forgot to remove those unneeded lines. Thanks for seeing this

**Comment by rickyaristio** [comment]
Hey @tteschemacher , could you approve this?

**Review by tteschemacher — APPROVED** [review]
Sure. go ahead, please! Thanks for the fixes.

**Pull request merged**